header-rewriter/tests/simple1/simple1.c

```diff
@@ -38,13 +39,13 @@ void simple_destroy(struct Simple *s) { free(s); }
 
 void simple_foreach_v1(struct Simple *s, int (*map_fn)(int)) {
   for (;;) {
-    int value = (*s->scb.read_cb)(s->state);
+    int value = (s->scb.read_cb)(s->state);
```

Comment:
Parentheses are optional here.

Reply:
We're not rewriting fn ptrs in this file anymore (it has pkey 0) so it's not relevant.